### PR TITLE
Update website to work with Node 10

### PR DESCRIPTION
**Pull request by NickCuso**
# Description

Before this PR, attempting to run `npm i` would fail if you have Node 10 and not Node 8 specifically.  The package change allows `npm i` to execute.

Then `npm run dev` fails with a callback error.  Switching to use `writeFileSync` corrects that issue.

Now it works for me, even on Windows \o/

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [x] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol.com/package.json [hunk 1]
```diff
@@ -46,10 +46,6 @@
     "@svgr/cli": "4.3.2",
     "nodemon": "1.19.2"
   },
-  "engines": {
-    "node": "8.16.1"
```
Comment (julien51) [inline unlock-protocol.com/package.json]: I think we need to keep a hardcoded version in there so we have some "locked" version to make sure things run well!

Comment (NickCuso) [reply unlock-protocol.com/package.json]: hmm, can we lock it to the current LTS version then?  10.16.3

**Comment by cnasc** [comment]
I support this, and I agree that we should put the latest LTS in the `engines`.

As an aside, I've been running with v10 for the last few days and haven't noticed any problems, though I haven't run all the apps with it. We should update all the `package.json`s at the same time.

**Comment by cnasc** [comment]
(and if you run into any issues updating the other apps, I'm more than happy to help remediate)

**Comment by NickCuso** [comment]
Updated packages but have not tested the other apps yet. 

I've tried unlock-protocol.com and smart-contracts so far.

**Comment by NickCuso** [comment]
`unlock-core.dockerfile` starts with `FROM node:8.16.1-alpine`, I assume we should update this as well but unsure where the `-alpine` came from..

**Comment by cnasc** [comment]
> `unlock-core.dockerfile` starts with `FROM node:8.16.1-alpine`, I assume we should update this as well but unsure where the `-alpine` came from..

https://github.com/nodejs/docker-node/blob/a9c583095d4cf08bbd68f570a1f9a99780820351/10/alpine/Dockerfile

I think you just have to update it with the relevant Alpine Linux based node dockerfile

**Comment by NickCuso** [comment]
Ah that's a linux flavor.  Okay updated.

Little worried about CI's fail fast here.  Anyway to force all tests to run?

**Comment by cnasc** [comment]
> Ah that's a linux flavor. Okay updated.
> 
> Little worried about CI's quick-pass here. Anyway to force all tests to run?

Looks like it's running ok, what do you mean?

**Comment by NickCuso** [comment]
```
#!/bin/bash -eo pipefail
scripts/monorepo.sh unlock-js $CIRCLE_SHA1 $CIRCLE_BRANCH
No change in unlock-js, skipping job.
```

That's not much reassurance ;)

This is from `monorepo.sh` but I'm not much of a scripter.  Does not look like there is any workaround, but we should add one.  Is it possible to say something like `if title contains FORCE`?

**Comment by NickCuso** [comment]
Hmm alternatively I could add a `.forceBuild` file to each directory which contains a number.  Anytime we want to ensure tests are run for a certain app we increment the number.

**Comment by cnasc** [comment]
> ```
> #!/bin/bash -eo pipefail
> scripts/monorepo.sh unlock-js $CIRCLE_SHA1 $CIRCLE_BRANCH
> No change in unlock-js, skipping job.
> ```
> 
> That's not much reassurance ;)
> 
> This is from `monorepo.sh` but I'm not much of a scripter. Does not look like there is any workaround, but we should add one. Is it possible to say something like `if title contains FORCE`?

🤔 Hm, unlock-js doesn't have any files changed in this PR, but it looks like tests were also skipped for projects that do

**Comment by NickCuso** [comment]
That part is cause it's based on the last specific commit, it should probably change to compare to master instead.

My concern is that this rule considers the change to docker/unlock-core.dockerfile a no-op.

**Comment by cnasc** [comment]
> My concern is that this rule considers the change to docker/unlock-core.dockerfile a no-op.

I see what you mean. Let me take a look at the script

**Comment by cnasc** [comment]
It looks like we don't have direct access to the PR title there, and I'm not familiar enough with the surrounding CI context to know how to easily do it.

One option we have is to merge, since CI always runs all tests on master builds. If we see any failures we can back them out quickly.

**Pull request merged**